mango/container/core.py

```diff
@@ -621,9 +124,24 @@ def register_agent(self, agent, suggested_aid: str = None):
             self._no_agents_running = asyncio.Future()
         aid = self._reserve_aid(suggested_aid)
         self._agents[aid] = agent
+        agent._do_register(self, aid)
         logger.debug("Successfully registered agent;%s", aid)
         return aid
 
+    def include(self, agent: A, suggested_aid: str = None) -> A:
```

Comment:
Is there a particular reason this is called `include` now? It's still `register` in Mango.jl so mildly confusing.

Reply:
I will change register_agent to register, and delete include. The idea was to keep register_agent unbroken, but in fact nobody used manual registering anyway as it was included in the Agent constructor.